src/rpc.rs

```diff
@@ -321,6 +324,11 @@ impl Connection {
     }
 
     fn blockchain_openassets_scripthash_listunspent(&self, params: &[Value]) -> Result<Value> {
+        if !self.enable_open_assets {
+            bail!(ErrorKind::OpenAssetsError(
+                "open assets feature is disabled".to_owned()
+            ));
+        }
```

Comment:
I think it is possible to check the validity of these functions in `handle_command`, and return the `unknown method` error if the functions are not valid?

Reply:
c0f160fd71010f62f81f8eb954ca2032d857146f

Reply:
I fixed the implementation to use `match guard`.(98934cf)